contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
I was sure that `service` command is still used in CentOS7 for backward compatibility with `centos6`, it just redirects `service` -> `systemctl`.

For example my Vagrant CentOS7 has `service` command preinstalled: 
```sh
cat /etc/centos-release
CentOS Linux release 7.2.1511 (Core)

[root@centos7 vagrant]# service
Usage: service < option > | --status-all | [ service_name [ command | --full-restart ] ]
```